Emma/emma_libs/memoryMap.py

```diff
@@ -271,7 +265,7 @@ def createReportPath(outputPath, projectName, reportName, fileExtension):
     :return: The created path string.
     """
     Emma.shared_libs.emma_helper.mkDirIfNeeded(outputPath)
-    memStatsFileName = projectName + "_" + reportName + "_" + TIMESTAMP + "." + fileExtension
+    memStatsFileName = projectName + "_" + reportName + "_" + TIMESTAMP + fileExtension
```

Comment:
Extension should be without dot so I think the original line was correct
```suggestion
    memStatsFileName = projectName + "_" + reportName + "_" + TIMESTAMP + "." + fileExtension
```

Also add a note about this [in the docstring](https://github.com/bmwcarit/Emma/pull/55/commits/2cfdf3ef2b898d53a3d063e9717b2912f1b2acc6#diff-ebc0a54aa7b560ca1d5e013c871015bfR264):
```
    :param fileExtension: [str] the name of file extension (without dot -> "csv" NOT ".csv")
```